opengrok-indexer/src/main/java/org/opengrok/indexer/history/GitRepository.java

```diff
@@ -57,7 +57,7 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(GitRepository.class);
 
-    private static final long serialVersionUID = 1L;
+    private static final long serialVersionUID = -6126297612958508386L;
```

Comment:
why is such an ugly number necessary?

Reply:
This is what IDEA generated (which I think is based on the default Java algorithm).

Reply:
ok